### ISSUE TITLE
local page loading slowly, local theme page doesn't show pictures.

### ISSUE THREAD

**Issue by outerpreneur** [Shopify/shopify-cli#1974]
# Issue summary

Write a short description of the issue here ↓

local page loading slowly, local theme page doesn't show pictures.

## Expected behavior

What do you think should happen? load normally with all pictures

## Actual behavior

What actually happens?

Tip: include an error message (in a `<details></details>` tag) if your issue is related to an error

local page not loading properly since I updated.

## Steps to reproduce the problem

1. shopify theme serve
1. go to the local page for live local preview http://127.0.0.1:9292/
1.page loads slowly and with no pictures

## Specifications

- App type: Shopify CLI 
- Operating System: Macos Monterrey 12.2
- Shell: Homebrew
- Ruby version (ruby -v):ruby 2.6.8p205

**Comment by outerpreneur** [comment]
> I got the same issue after upgrading this morning, with images not loading. was you able to get it working?

Not yet!

**Comment by mitchuman** [comment]
Same issue, broken/missing CSS in certain areas on v2.10.0

**Comment by sb18281** [comment]
Similar issue,

Found it was an auto fade in JS lib that had been failing. 

Which lead to finding a network error with theme.js (_ERR_CONTENT_LENGTH_MISMATCH 200_). Likely that this may be happening with other files too.

Rolling back to v2.9.0 or earlier fixes the issue

**Comment by mitchuman** [comment]
How do you rollback using `brew`?

**Comment by cardfaux** [comment]
> How do you rollback using `brew`?

I have been trying to figure this out for the past couple of hours, have you had any luck rolling it back to 2.9.0?

**Comment by ankitshekhawat** [comment]
For now I just resorted to using a gem instead of  a brew `gem install shopify-cli -v 2.9.0`

**Comment by sharafciger** [comment]
Seems like they broke some assets pathing that were cached with this update. 
I managed to make it work by deleting all local files and pulling them again from the store before serve command.

**Comment by outerpreneur** [comment]
> Can also confirm this is an issue with v2.10.0. Rolling back to v2.9.0 or earlier fixes the problem.

How do you roll back? Can it be it done using brew?

**Comment by vfxjohow** [comment]
Rolled back with:

`gem uninstall shopify-cli `  -> deleted latest version
`gem install shopify-cli -v 2.9.0`  -> reinstall previous version

This solved the problem for now to keep up work for today.

**Comment by binarymonkey84** [comment]
Yep, also just had this problem and sank a lot of time into it thinking it was something I'd done.

I'm on Windows 10 + Chrome using the gem package.

Rolling back...

**Comment by outerpreneur** [comment]
New version 2.10.1 is out and working fine
